### ISSUE TITLE
提取特征那一步有问题

### ISSUE THREAD

**Issue by jzbbobo** [willard-yuan/flask-keras-cnn-image-retrieval#1]
在提取特征那里，您好像是获取了1000维的score作为特征并不是4096维的特征。希望您看下！

**Comment by willard-yuan** [comment]
@a1311543690 我当时测试的时候好像是提取的4096维，我晚上再确认一下。

**Comment by willard-yuan** [comment]
@a1311543690 See [63](https://github.com/willard-yuan/flask-keras-cnn-image retrieval/blob/master/extract_CNN_VGG_Keras.py#L63) line:

``` python
 #Remove the last two layers to get the 4096D activations
model.layers.pop()
model.layers.pop()
```

Extract the 4096 dimension activations from the VGGNet built in Keras. The last two layer have been cut off - The last Softmax Layer and Dropout layer.
